### PR TITLE
Hotfix dieFounder,fragen,mainborn and hero sections

### PR DESCRIPTION
**Pull request by NoellaPromise**
In this PR, we worked on dieFounder,fragen,mainborn and hero sections

### CODE REVIEW

File: src/sections/hero.html [hunk 1]
```diff
@@ -27,27 +25,27 @@ <h1 class="text-sm text-white lg:text-5xl">Europas <span class="text-green">Digi
...
+            <span class="w-28 h-28"><img src="../../images/shakingHandsIcon.svg" alt="Shaking hands icon"></span>
               <p class="text-xs "><span class=" text-green font-railwayBold">Talent & Quality</span> <br>
                 Viele junge, gut ausgebildete und 
                  motivierte Menschen 
             </p>
         </div>
         <div class="flex flex-row py-6 space-x-6 mr-28">
             <span class="w-28 h-28"><img src="../../images/locationIcon.svg" alt="Location icon"></span>
-              <p class="text-xs "> <span class=" text-green font-railwayBold">Location</span> <br>
+              <p class="text-xs"> <span class="text-green font-railwayBold">Location</span> <br>
```
Comment (NelHim) [inline src/sections/hero.html]: There are still break tags. Kindly remove them

**Review by NelHim — CHANGES_REQUESTED** [review]

File: src/sections/hero.html [hunk 2]
```diff
@@ -27,27 +25,27 @@ <h1 class="text-sm text-white lg:text-5xl">Europas <span class="text-green">Digi
...
+        <div class="lg:mt-[90px] sm:w-[800px] sm:h-[500px] relative -bottom-14"><img src="../../images/Ojembees.webp" alt="Picture of the Ojembees" ></div>  
       </div>
       </section>
 
-      <section class="relative z-10 flex-row text-white lg:relative lg:flex ojemba-gradient1 font-railwayLight lg:gap-48 lg:p-x-12">
+      <section class="relative z-10 flex-row text-white lg:relative lg:flex ojemba-gradient1 font-railwayLight lg:gap-48 lg:p-x-12 xl:h-28">
         <div class="flex flex-row pt-4 gap-x-2 text-xs lg:space-x-6 lg:py-6 lg:ml-44">
-            <span class="w-28 h-28 "><img src="../../images/handIcon.svg" alt="Hand Icon"></span>
+            <span class="w-28 h-28"><img src="../../images/handIcon.svg" alt="Hand Icon"></span>
               <p class="text-xs"> <span class=" text-green font-railwayBold">Kostengünstig</span> <br>
```
Comment (divinecharlotte) [inline src/sections/hero.html]: - [ ] kindly remove the below br tags too 😄

File: src/sections/hero.html [hunk 3]
```diff
@@ -27,27 +25,27 @@ <h1 class="text-sm text-white lg:text-5xl">Europas <span class="text-green">Digi
...
              niedriges Lohnkostenniveau im europäischen Vergleich
                  </p>
         </div>
         <div class="flex flex-row py-6 space-x-6">
-            <span class="w-28 h-28" ><img src="../../images/shakingHandsIcon.svg" alt="Shaking hands icon"></span>
+            <span class="w-28 h-28"><img src="../../images/shakingHandsIcon.svg" alt="Shaking hands icon"></span>
               <p class="text-xs "><span class=" text-green font-railwayBold">Talent & Quality</span> <br>
```
Comment (divinecharlotte) [inline src/sections/hero.html]: - [ ] kindly remove also the below br tags too 😄

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
requested some changes kindly fix them and you will be good to go

**Review by divinecharlotte — APPROVED** [review]
looks good now 👏

**Review by NelHim — APPROVED** [review]
Good Job

**Pull request merged**